src/plugin_system/testing/render_service.py

```
@@ -0,0 +1,96 @@
+        display_manager=display_manager,
+        cache_manager=cache_manager,
+        plugin_manager=plugin_manager,
+        install_deps=False,
+    )
+
+    start_time = time.time()
+
+    if not skip_update:
+        try:
+            plugin_instance.update()
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)